Flutter/datagrid/filtering.md

```diff
@@ -1325,4 +1325,86 @@ Widget build(BuildContext context) {
 {% endhighlight %}
 {% endtabs %}
 
-<img alt="flutter datagrid shows filter icon when hover the header cell" src="images/filtering/flutter-datagrid-show-filter-icon-on-hover.gif"/>
\ No newline at end of file
+<img alt="flutter datagrid shows filter icon when hover the header cell" src="images/filtering/flutter-datagrid-show-filter-icon-on-hover.gif"/>
+
+## Set a StrongDataType for filter behavior
+The [FilterBehavior](https://pub.dev/documentation/syncfusion_flutter_datagrid/latest/datagrid/FilterBehavior.html) property specifies whether filtering should consider a cell's value as a string or as a data type.When using the StrongDataType option,the comparison is made directly to their data types.

```

Comment:
By default, the `FilterBehavior.strongDataType` applies to the num, string, and DateTime types. Now, it also applies to the user-defined types also. It can be enabled by extending the user-defined type with the `Comparable` class.

Reply:
Modified the content